### PR TITLE
feat(editing-support): add stickybuf.nvim

### PR DESCRIPTION
**Pull request by jay-babu**
## 📑 Description

add stickybuf.nvim

### CODE REVIEW

**Comment by Uzaaft** [comment]
@luxus  You could just complete the review next time :)

File: lua/astrocommunity/editing-support/stickybuf-nvim/init.lua [hunk 1]
```diff
@@ -0,0 +1,5 @@
+return {
+  "stevearc/stickybuf.nvim",
+  opts = {},
+  event = { "User AstroFile", "InsertEnter" },
```
Comment (Uzaaft) [inline lua/astrocommunity/editing-support/stickybuf-nvim/init.lua:4]: Won't `User AstroFile` be enough?

Comment (Uzaaft) [reply lua/astrocommunity/editing-support/stickybuf-nvim/init.lua:4]: For comparasion, stevearc does:
`  { "stevearc/stickybuf.nvim", cmd = { "PinBuffer", "PinBuftype", "PinFiletype" }, opts = {} },
`

**Comment by jay-babu** [comment]
> @luxus  You could just complete the review next time :) 

It would only let me check off one box. Not all for some reason

**Comment by Uzaaft** [comment]
> > @luxus  You could just complete the review next time :)
> 
> It would only let me check off one box. Not all for some reason

You dont appear as the one Who edited the checkbox😂🤷‍♂️

File: lua/astrocommunity/editing-support/stickybuf-nvim/init.lua (hunk 1 above)
Comment (jay-babu) [inline lua/astrocommunity/editing-support/stickybuf-nvim/init.lua:4]:
```suggestion
  event = { "User AstroFile" },
  cmd = { "PinBuffer", "PinBuftype", "PinFiletype" },
```

**Comment by jay-babu** [comment]
@Uzaaft @luxus should be good now. thanks

**Review by Uzaaft — APPROVED** [review]

**Pull request merged**